shell/app/modules/workBench/pages/projects/project-list-protocol.tsx

```diff
@@ -16,13 +16,33 @@ import DiceConfigPage from 'config-page/index';
 import { useUpdate } from 'common';
 import { get } from 'lodash';
 import ApplyUnblockModal, { IMetaData } from 'workBench/pages/projects/apply-unblock-modal';
+import routeInfoStore from 'app/common/stores/route';
 
-export const ProjectList = () => {
+const ProjectList = () => {
   const [state, updater, update] = useUpdate({
     visible: false,
     metaData: {} as IMetaData,
+    scenarioKey: '',
+    scenarioType: '',
```

Comment:
actually, u can initialize state here and remove useEffect, the problem of using `useEffect` is the first time page render these two state would be empty.
```
scenarioKey: currentRoute.relativePath === 'public-projects' ? 'project-list-all' : 'project-list-my',
scenarioType: currentRoute.relativePath === 'public-projects' ? 'project-list-all' : 'project-list-my',
````